week2/index.html

```diff
@@ -0,0 +1,47 @@
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title >My First HTML</title>
+        <nav class="menu">
+            <ul>
+              <li><a href="#">Home</a></li>
+              <li><a href="#">About</a></li>
+              <li><a href="#">Contact</a></li>
+            </ul>
+          </nav>
+    </head>
+    <header>
```

Comment:
The structure of html page is:
&lt;!DOCTYPE html&gt;
&lt;html lang="en"&gt;
&lt;head&gt;
&nbsp;&nbsp;&nbsp;&nbsp;&lt;title&gt;Title of page&lt;/title&gt;
&lt;/head&gt;
&lt;body&gt;
&nbsp;&nbsp;&nbsp;&nbsp;HTML page content
&lt;/body&gt;
&lt;/html&gt;

As &lt;header&gt; is part of your HTML page, it should be inside &lt;body&gt;